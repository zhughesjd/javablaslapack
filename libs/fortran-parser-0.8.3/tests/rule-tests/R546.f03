! testing target statement, R546
10 target :: i
target j, A(10,20)
end
