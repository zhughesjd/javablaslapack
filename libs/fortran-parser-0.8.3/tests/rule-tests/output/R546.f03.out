R1101:main-program__begin:
R566-F2008:target-decl: id=[@8,46:46='i',<206>,2:13] hasArraySpec=false hasCoArraySpec=false
R 546:target-stmt: label=[@2,33:34='10',<10>,2:0] count=1
R566-F2008:target-decl: id=[@12,55:55='j',<206>,3:7] hasArraySpec=false hasCoArraySpec=false
R 406:int-literal-constant: digitString=[@17,60:61='10',<10>,3:12] kindParam=null
R 510:array-spec-element: type=700
R 406:int-literal-constant: digitString=[@19,63:64='20',<10>,3:15] kindParam=null
R 510:array-spec-element: type=700
R 510:array-spec: count=2
R566-F2008:target-decl: id=[@15,58:58='A',<206>,3:10] hasArraySpec=true hasCoArraySpec=false
R 546:target-stmt: count=2
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=2
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
